jsk_perception/node_scripts/ssd_object_detector.py

```diff
@@ -120,6 +124,16 @@ def image_cb(self, msg):
         bboxes, labels, scores = self.model.predict([img])
         bboxes, labels, scores = bboxes[0], labels[0], scores[0]
 
+        msg_indices = ClusterPointIndices(header=msg.header)
```

Comment:
`msg_indices` is not friendly to developers because there is `indices_msg`.
How about `msg_indices` -> `cluster_indices_msg`?

Reply:
renamed